direct/io-hbase-bindings/src/main/java/cz/o2/proxima/direct/hbase/Util.java

```diff
@@ -35,18 +35,19 @@
 
   static Configuration getConf(URI uri) {
     Configuration conf = HBaseConfiguration.create();
+    List<String> paths = UriUtil.parsePath(uri);
+    if (paths.size() > 1) {
+      conf.set(HConstants.ZOOKEEPER_ZNODE_PARENT, paths.get(0));
```

Comment:
This looks like it should be more like `paths.get(0..size - 1)`, or am I wrong?

Reply:
Thank you! I completely miss this situation. Fixes and also added new assert in test.